releasenotes/notes/0.19/add-support-to-disable-amplitude-limit-in-parametric-pulses-ef88b77db8c1b06c.yaml

```diff
@@ -0,0 +1,10 @@
+---
+features:
+  - |
+    Added an argument ``limit_amplitude`` to the constructor of ``ParametricPulse``,
+    which is the base class of :obj:`.Gaussian`, :obj:`.GaussianSquare`, :obj:`.Drag` and :obj:`.Constant`, to
+    enable or disable the amplitude limit of 1 on the pulse. This
+    option gets propagated to the parent class ``Pulse``. With this option enabled,
+    the ``ParametricPulse`` class and its subclasses do not allow amplitudes larger
+    than 1. By default, the value of ``limit_amplitude`` is true, so that the
```

Comment:
I assume they raise an exception should we document what that is here?

Reply:
I rewrote this whole note - the note is pretty misleading.  In reality, the option lets you _disable_ the error (which already existed) on an instance-by-instance basis.